src/olympia/devhub/views.py

```diff
@@ -661,7 +662,7 @@ def file_validation(request, addon_id, addon, file_id):
 
     validate_url = reverse('devhub.json_file_validation',
                            args=[addon.slug, file_.id])
-    file_url = reverse('files.list', args=[file_.id, 'file', ''])
+    file_url = code_manager_url('browse', addon.pk, file_.version.pk)
```

Comment:
It looks like this change should be made, but I'm not sure what effect it has on the site or where to test it. After making this change no tests broke, so I assume this line is not currently covered by tests.

Reply:
so maybe we shouldn't touch that in this patch?